api/src/main/java/org/hawkular/alerts/api/services/AlertsCriteria.java

```diff
@@ -293,31 +295,27 @@ public void setThin(boolean thin) {
     }
 
     public boolean hasAlertIdCriteria() {
-        return null != alertId
-                || (null != alertIds && !alertIds.isEmpty());
+        return null != alertId || !isEmpty(alertIds);
     }
 
     public boolean hasSeverityCriteria() {
-        return null != severity
-                || (null != severities && !severities.isEmpty());
+        return null != severity || !isEmpty(severities);
     }
 
     public boolean hasStatusCriteria() {
-        return null != status
-                || (null != statusSet && !statusSet.isEmpty());
+        return null != status || !isEmpty(statusSet);
```

Comment:
-1
Same case, status is an Status object, so isEmpty() is just wrapping the null validation and hiding this.